### PR TITLE
Fixed admin kicking

### PR DESCRIPTION
**Pull request by Johnnybeefcake**

### CODE REVIEW

File: Initium-Odp/src/com/universeprojects/miniup/server/HtmlComponents.java [hunk 1]
```diff
@@ -468,10 +468,10 @@ public static String generateGroupMember(CachedEntity viewingChar, CachedEntity
 				}
 			}
 			
-			if (groupStatus.equals("Kicked") == false)
-				sb.append("<a onclick='groupMemberKick(event, " + character.getKey().getId() + ")'>Kick</a>");
-			else
-				sb.append("<a onclick='groupMemberKickCancel(" + character.getKey().getId() + ")'>Cancel Kick</a>");
+			if(groupStatus.equals("Kicked") == false && isAdmin == false || viewCreator)
```
Comment (Emperorlou) [inline Initium-Odp/src/com/universeprojects/miniup/server/HtmlComponents.java]: Better to wrap the or logic in braces to make it clear what is intended.

File: Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandGroupMemberKick.java [hunk 2]
```diff
@@ -1,88 +1,95 @@
-package com.universeprojects.miniup.server.commands;
```
Comment (Emperorlou) [inline Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandGroupMemberKick.java]: This entire file is shown as different. It seems to happen sometimes but I don't know why. I'll need to see a proper diff before I can approve. Try reverting and making your changes again?

File: Initium-Odp/war/odp/ajax_group.jsp [hunk 3]
```diff
@@ -258,7 +258,10 @@
 								+ ")'>Demote from admin</a>");
 					}
 				}
-				if (groupStatus.equals("Kicked") == false)
+				if (groupStatus.equals("Kicked") == false 
+						&& groupStatus.equals("Admin") == false 
+						|| (Key) group.getProperty("creatorKey")).getId() ==
```
Comment (Emperorlou) [inline Initium-Odp/war/odp/ajax_group.jsp]: Key equal tests should be done using GameUtils.equals()

Also please wrap your or logic in braces to indicate clearly what the intention is here.

Comment (Johnnybeefcake) [inline Initium-Odp/war/odp/ajax_group.jsp]: This file's no longer being used so changes to it should be discarded. Made inferior by ajax_group.jsp in odppages

File: Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandGroupMemberKick.java [hunk 4]
```diff
@@ -77,6 +77,13 @@ public final void run(final Map<String, String> parameters)
 		{
 			throw new UserErrorMessage("The creator cannot be kicked you fool.");
 		}
+		
+		if ("Admin".equals(admin.getProperty("groupStatus")) == true
```
Comment (Emperorlou) [inline Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandGroupMemberKick.java]: I believe creators have their groupStatus as Admin as well.

**Comment by Emperorlou** [comment]
Closed due to being no longer relevant.

**Pull request closed without merging**